distributed/hdfs.py

```diff
@@ -283,3 +284,58 @@ def write_bytes(path, futures, executor=None, hdfs=None, **hdfs_auth):
         filenames = [os.path.join(path, template % i) for i in range(n)]
 
     return executor.map(write_block_to_hdfs, filenames, futures, hdfs=hdfs)
+
+
+@gen.coroutine
+def _read_text(fn, encoding='utf-8', errors='strict', lineterminator='\n',
```

Comment:
errors='strict' fairly reflects decode usage, but I expect 'ignore' to be used more commonly.